tor/core/__init__.py

```diff
@@ -57,5 +57,5 @@ def __get__(self, obj, _type=None):
 ]
 
 CLAIM_PHRASES = ["claim", "dibs"]
```

Comment:
I think this would be a good time to also add some common `claim` mistypes, such as `clam` and `caim` and `clai`.

Reply:
yep, good call